app/src/main/java/org/mozilla/fenix/HomeActivity.kt

```diff
@@ -264,6 +265,7 @@ open class HomeActivity : LocaleAwareAppCompatActivity(), NavHostActivity {
             attachOnHomeActivityOnCreate(lifecycle)
         }
 
+        NavGraphProvider.blockForNavGraphInflation(navHost.navController)
```

Comment:
(same comment) Why are these new lines necessary? In the initial PR, we added lines like these but they were unnecessary and only ended up slowing down the browser unnecessarily in some cases.

Reply:
I'll remove.  Thanks